scripts/audit_branches.sh

```
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Get all remote branches excluding main and HEAD
+branches=$(git branch -r | grep -v "HEAD" | grep -v "origin/main" | sed 's/origin\///')
+
+for branch_name in $branches; do
+  branch="origin/$branch_name"
+  echo "--- Branch: $branch ---"
+
+  # Check files
+  files=$(git ls-tree -r "$branch" --name-only 2>/dev/null)
+  if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)